app/controllers/cities_controller.rb

```diff
@@ -76,10 +76,8 @@ class CitiesController < ApplicationController
     ["Bourges", "18000"]
   ]
   # Separate names and postal codes
-  MAIN_NAMES = []
-  MAIN_CITIES.each{ |city| MAIN_NAMES.push(city[0]) }
-  MAIN_POSTAL_CODES = []
-  MAIN_CITIES.each{ |city| MAIN_POSTAL_CODES.push(city[1]) }
+  MAIN_NAMES = MAIN_CITIES.map(&:first)
```

Comment:
Unnecessary now

Reply:
I will keep the name and postal codes list as it would be pretty unreadable with only postal codes and it could be useful in the near future.
Better option is to have population in DB.